src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator/Calculator';
 
 function App() {
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/164142978-7b99c87e-5d8a-4511-9a18-c8511faa39e6.png)

Therefore:

- [ ] Please, refactor this component to a class-based component (App is a component too).  
That way, we will meet this particular project requirement and we will move closer to approval. 💪 

## Explanation

In order to meet the requirements, all the components should be class-based.  `App` is a component too.